packages/repo/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_VERSION=2.20
+TERMUX_PKG_HOMEPAGE=https://android.googlesource.com/tools/repo
+TERMUX_PKG_DESCRIPTION="The Multiple Git Repository Tool from the Android Open Source Project"
+TERMUX_PKG_LICENSE="APACHE"
+TERMUX_PKG_LICENSE_FILE="LICENSE"
+TERMUX_PKG_MAINTAINER="@termux"
```

Comment:
Could you keep the correct order of variables?

And `APACHE` is not a valid SPDX license identifier. `Apache-2.0` should be specified in `TERMUX_PKG_LICENSE`.

Reply:
Correct field order sample: https://github.com/termux/termux-packages/wiki/Creating-new-package#writing-a-package-script. Dependencies go below SHA-256.

Just to note that scripts sometimes bulk-modified by script and mixed field order is headache for maintainers.